### ISSUE TITLE
[Feature Request] Self Destruct Photo/Video in Normal Private chats

### ISSUE THREAD

**Issue by AhyaH-YHWH** [telegramdesktop/tdesktop#9780]
Since Self Destruct Photo/Video (With Caption) can be sent in Normal Private Chats too

without implementing Secret Chat,
It is possible to add this Feature

At least , Windows 10 has ability to make screenshot of App Black, when, It is blocked by app
(I have seen Unigram Screenshot of Secret chats, turning Black, effectively disabling screenshot)

Currently, it shows the message, I got a self destruct Photo/Video on Desktop 2.4.11 and I should see it in Android/iOS or Unigram Windows 10

**Comment by vn971** [comment]
This is how Self-Destruct looks in a **non-secret** chat in Android:
<img src="https://user-images.githubusercontent.com/1547617/103487774-2016c480-4e08-11eb-9ebd-d603c6b9d6dc.png" width="400em" />

**Comment by vn971** [comment]
@stale yes the issue is still there, we want self-destruct support in normal chats, same as in a normal (non-encrypted) chat on Android.

**Comment by soredake** [comment]
Not stale

**Comment by soredake** [comment]
Not stale

**Comment by soredake** [comment]
Not stale.

**Comment by Aokromes** [comment]
> > enhancements aren't target of slate bot.
> 
> But it's closing it constantly?

like you see, the last time slate bot aftected this group was 2 years ago, now slate bot doesn't target enhancements

**Comment by Twilightssuperb** [comment]
Still no news?

We need just sending some media from PC, not viewing, OK. Now I have to send file in Saved etc., then open mobile TG client to download such media and send with timer; it's very annoying.

Initial version could implement just sending, please. 🙏
